contribs/gmf/src/directives/partials/contextualdata.html

```diff
@@ -0,0 +1,2 @@
+This is the default template for contextual data.
+This should be overridden in applications.
```

Comment:
Is it relay needed?
I ask it because it's loaded in the applications and it's never used ...